src/annotation_tool/project.py

```diff
@@ -129,27 +131,20 @@ def set_tsv_file(self, to : str):
         self.tsv_file = to
         if not os.path.exists(self.tsv_file):
             return
-        with open(self.tsv_file) as file:
-            first = True
-            for line in file:
-                if first:
-                    first = False
-                    continue
+        with open(to, newline='') as file:
```

Comment:
Why did you specify `newline=''` ?

Reply:
https://docs.python.org/3/library/csv.html#csv.reader

> If csvfile is a file object, it should be opened with newline=''. [[1]](https://docs.python.org/3/library/csv.html#id3)